sentry-ruby/lib/sentry/session.rb

```diff
@@ -17,9 +16,11 @@ def initialize
       @aggregation_key = Time.utc(@started.year, @started.month, @started.day, @started.hour, @started.min)
     end
 
-    # TODO-neel add :crashed after adding handled mechanism
-    def update_from_exception(_exception = nil)
+    def update_from_event(event)
       @status = :errored
```

Comment:
Should this be moved behind the return below? Since we name it `update_from_event` now, setting the session to `errored` when it's called with any event is weird (e.g. technically `CheckInEvent` can also be passed here.)

IMO we should probably name it `update_from_exception_event` unless we plan to expand its usage for other event types too.

Reply:
done